### PR TITLE
Doc 10604  performance page proofreading

### PR DESCRIPTION
**Pull request by RichardSmedley**

### CODE REVIEW

**Comment by RichardSmedley** [comment]
Further improvements to this page will need a refactoring - planned for a future SDK release, in a broader context of SDK best practice discussions.

File: modules/project-docs/pages/performance.adoc [hunk 1]
```diff
@@ -224,7 +228,7 @@ The SDK by default enables operation tracing and metrics tracking.
 It is used to generate threshold and orphan response reports which are written to the log file. 
 While these are useful tool for debugging, they do come at a cost of increased memory and CPU use. 
 Operation tracing and metrics can be disabled by setting the `ClusterOptions.TracingOptions.Enabled` flag to `false` and/or by setting the `ClusterOptions.LoggingMeterOptions.Enabled` to `false`. 
-Do not by doing so you will lose the ability to use these useful debugging tools.
+Note, by doing so you will lose the ability to use these useful debugging tools.
```
Comment (tom-rosewell) [inline modules/project-docs/pages/performance.adoc:231]: Should this be an admonition?

**Review by tom-rosewell — APPROVED** [review]
LGTM

File: modules/project-docs/pages/performance.adoc (hunk 1 above)
Comment (RichardSmedley) [reply modules/project-docs/pages/performance.adoc:231]: Maybe when we refactor  :+1:

File: modules/project-docs/pages/performance.adoc [hunk 2]
```diff
@@ -62,7 +64,7 @@ var tasks = Enumerable.Range(1, 100).Select(i => collection.GetAsync($"key{i}"))
 var results = await Task.WhenAll(tasks).ConfigureAwait(false);
 ----
 
-When done in this manner, the keys and response will be sent and recieved in out-of-order manner: "key2, "key10", "key3"...
+When done in this manner, the keys and response will be sent and recieved out-of-order: "key2, "key10", "key3"...
```
Comment (osfameron) [inline modules/project-docs/pages/performance.adoc:67]: received

Comment (osfameron) [reply modules/project-docs/pages/performance.adoc:67]: "key2"

**Pull request merged**